bigframes/session/_io/bigquery/read_gbq_table.py

```diff
@@ -152,24 +152,23 @@ def validate_table(
     return False
 
 
-def are_index_cols_unique(
+def infer_primary_key(
```

Comment:
Since this could return `index_cols` or the primary keys, perhaps a different name would be better?

```suggestion
def infer_unique_columns(
```